### ISSUE TITLE
Can authenticate but can NOT attach to token to header

### ISSUE THREAD

**Issue by yankeeinlondon** [a85/POSTMan-Chrome-Extension#1449]
I have specified that a given endpoint on FitBit requires OAuth authentication and then gone through the steps of adding an access token:

![image](https://cloud.githubusercontent.com/assets/980757/10067671/4e99a6f0-6296-11e5-9693-5f359ad83f10.png)

I then make the request to the endpoint and get the following error:

```
{
  "errors": [
    {
      "errorType": "oauth",
      "fieldName": "n/a",
      "message": "No Authorization header provided in the request. Each call to Fitbit API should be OAuth signed"
    }
  ],
  "success": false
}
```

I sent in an email to support and got back a response that I'm meant to click on the now listed Access Token:

![image](https://cloud.githubusercontent.com/assets/980757/10067738/b8c3308c-6296-11e5-914d-3226b3e25939.png)

Unfortunately no amount of clicking changes the outcome.

**Comment by abhijitkane** [comment]
The Fitbit API seems to have a little quirk in the format of the OAuth flow. A number of users (not Postman) have reported this - 
https://groups.google.com/forum/#!topic/fitbit-api/Fk7yOWqHhB8, https://groups.google.com/forum/#!topic/fitbit-api/KKEZhAQKXeQ

We'll look into ways of solving this, but Postman's OAuth2 flow caters to many services, and we'll have to make sure that those flows don't break.

**Comment by yankeeinlondon** [comment]
neither of these links is helpful. 

The first one appears to be invalid unless I'm missing something. The main conclusion is that rather than sending the token as a GET parameter you must send in the HTTP header. This is very common and you appear to have a setting for this:

![image](https://cloud.githubusercontent.com/assets/980757/10068048/26f065fa-6299-11e5-8139-cfd9f1a3d9bc.png)

So based on that thread I think we can exclude that. The second link wanders around but doesn't come to any conclusions. I'm not saying there isn't something unusual in the OAuth 2 handling of FitBit's (we all know that OAuth2's long standardisation led to some variations) but these two links are not going to explain or help anyone.

**Comment by abhijitkane** [comment]
The error message points to an error in the request made to get the access token (POST https://api.fitbit.com/oauth2/token).

The "Add token to header" option is used to add the access token (which we haven't been able to receive for Fitbit) to the actual request you want to make.

**Comment by yankeeinlondon** [comment]
Is this going to be resolved? FitBit is the most popular wearable in the world. Seems important. If not, is there a work-around?

**Comment by a85** [comment]
@ksnyde Agree. Give us some time to look into this in more detail.

**Comment by yankeeinlondon** [comment]
@a85 I just had the same experience with Google Calendar API. Is that known about too?

**Comment by SamvelRaja** [comment]
Tracked and fixed in the issue #1980
